sx-auth.el

```diff
@@ -38,14 +38,16 @@
   "http://vermiculus.github.io/sx.el/auth/auth.htm")
 (defconst sx-auth-client-id
   "3291")
-(defvar sx-auth-access-token
-  nil
+
+(defstash sx-auth-access-token
+    "auth.el" sx nil
```

Comment:
The latter option seems good, but what else would `auth` be used for (in our case specifically)?